### ISSUE TITLE
Make network objects round-trippable from API

### ISSUE THREAD

**Issue by jathanism** [dropbox/nsot#36]
Right now to create a network object you send this:

```http
{
    "attributes": {
        "hostname": "foo-bar1"
    },
    "cidr": "192.168.0.0/25"
}
```
But the API returns it like this:

```http
{
    "is_ip": false,
    "site_id": 1,
    "network_address": "192.168.0.0",
    "parent_id": 6,
    "prefix_length": 25,
    "ip_version": "4",
    "attributes": {
        "hostname": "foo-bar1"
    },
    "id": 7
}
```
We should make it where an object that comes out from the API could be used to create the object, too.

**Comment by coxley** [comment]
Is this completed with the latest PR?

**Comment by jathanism** [comment]
Fixed in #156
